api/AbsoluteOrientationSensor.json

```diff
@@ -29,7 +29,7 @@
             "version_added": "56"
           },
           "safari": {
-            "version_added": null
+            "version_added": true
```

Comment:
This is the first change that remains after rebasing on top of https://github.com/mdn/browser-compat-data/pull/3700. AFAICT this IDL file isn't in WebKit at all, there's just LayoutTests/imported/w3c/web-platform-tests/interfaces/orientation-sensor.idl which is for testing.

I think you'll need to verify the paths you ingest and regenerate this PR.

Reply:
Can do, yeah!